Tests/feaLib/data/GSUB_5_formats.fea

```diff
@@ -1,16 +1,16 @@
 lookup GSUB5f1 {
-    ignore sub three four;
-    ignore sub four five;
+    ignore sub three' four';
```

Comment:
when we parse an ignore statement with no marked glyphs, we issue a warning and assume that only the first is marked; however when we convert it back to string with asFea() we put the apostrophe on all the glyphs, not just the first... I find this a bit odd. Shouldn't asFea also include a similar logic?

Reply:
Only the marked glyph (either explicitly or implicitly) is marked in `asFea()`. I marked all the glyphs here to keep the expected ttx output unchanged (previously we were marking all the glyphs). This reminds me, I need to add a new builder test for the implicit marking.

Reply:
> I marked all the glyphs here to keep the expected ttx output unchanged (previously we were marking all the glyphs)

but we don't want to keep the output unchanged, since we are no longer implicitly marking all the glyphs but only the first, no?

Reply:
I’m not sure what this test is testing, its name suggests it is testing various subtable formats of GSUB 5 lookup type, so I tried to keep the output unchanged since I presumed this is what it tests not the `ignore` statements.

Reply:
ok then I would rather fix the input test fea such that it no longer has ambiguously unmarked glyphs

Reply:
This is what I did already.

Reply:
oh I see, I got confused, sorry for the noise and thanks